Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM codeforafrica/hurumap:0.0.15
+FROM codeforafrica/hurumap:0.1.1
```

Comment:
QQ @phillipahereza, I don't see `v0.1.1` release/tag on https://github.com/CodeForAfrica/HURUmap, where did this come from?